Makefile

```diff
@@ -57,7 +57,7 @@ zlib-init:
 	mkdir -p $(LIB)
 	
 zlib-download: zlib-init
-	wget -c -O "$(ZLIB).tar.gz" "http://zlib.net/zlib-$(ZLIB_VERSION).tar.gz"
+	wget -c -O "$(ZLIB).tar.gz" "https://github.com/madler/zlib/archive/v$(ZLIB_VERSION).tar.gz"
```

Comment:
@Flamefire Why pointing to the github version instead of the official one ?

Reply:
I found the github one to work better, I had a couple CI failures due to it not finding the site, this reduces this to a single point of failure. Furthermore it eases future integration as a submodule or similar.